### PR TITLE
Improved README

### PR DESCRIPTION
**Pull request by JosXa**
- Changed wording in some spots
- Consistently applied rst markup
- Cleared up the examples with some comments

### CODE REVIEW

**Comment by elapouya** [comment]
Thank you for the readme. A little error nevertheless at line 21 : 
Could replace 

`obj = NoAttr()`

by 

`obj = NoAttr`

NoAttr is already an instance of NoAttrType as None is an instance of NoneType

Could you change your PR ?

File: README.rst [hunk 1]
```diff
@@ -2,37 +2,57 @@
...
+    >>> obj.a.b.c.
     AttributeError: 'NoneType' object has no attribute 'b'
     
-To avoid that, instead of returning a ‘None‘ value, one should return ‘NoAttr‘, by this way, 
-even next chained attribute will return ‘NoAttr‘ ::
+*NoAttr* gives you a way around this in cases where a series of ``None`` checks would be too much effort and result in complex code. 
+Instead of returning ``None``, you want to return an instance of ``NoAttr``, which will caused hierarchical attribute accesses to keep returning ``NoAttr`` until the end of the chain ::
+
+    >>> from noattr import NoAttr
+    >>> obj = NoAttr()
```
Comment (JosXa) [inline README.rst]:
```suggestion
    >>> obj = NoAttr
```

**Comment by JosXa** [comment]
Yep good catch! Done 👍

**Comment by elapouya** [comment]
Many thanks

**Pull request merged**